src/pages/my_page/MyPagePage.tsx

```diff
@@ -32,12 +32,78 @@ import ResearchProfile from '../research_profile/ResearchProfile';
 import { ProjectFormDialog } from '../projects/form/ProjectFormDialog';
 import { NavigationListAccordion } from '../../components/NavigationListAccordion';
 import NotFound from '../errorpages/NotFound';
+import { SelectableButton } from '../../components/SelectableButton';
+import { fetchTickets } from '../../api/searchApi';
+import { setNotification } from '../../redux/notificationSlice';
+import { TicketStatus } from '../../types/publication_types/ticket.types';
+
+const rowsPerPageOptions = [10, 20, 50];
+
+type SelectedStatusState = {
+  [key in TicketStatus]: boolean;
+};
+
+const StyledCheckbox = styled(Checkbox)({
+  paddingTop: '0.2rem',
+  paddingBottom: '0.2rem',
+});
 
 const MyPagePage = () => {
+  const dispatch = useDispatch();
   const { t } = useTranslation();
   const history = useHistory();
   const user = useSelector((store: RootState) => store.user);
   const queryClient = useQueryClient();
+  const [page, setPage] = useState(0);
+  const [rowsPerPage, setRowsPerPage] = useState(rowsPerPageOptions[0]);
+
+  const [selectedTypes, setSelectedTypes] = useState({
+    doiRequest: true,
+    generalSupportCase: true,
+    publishingRequest: true,
+  });
+
+  const [selectedStatuses, setSelectedStatuses] = useState<SelectedStatusState>({
+    New: true,
+    Pending: false,
```

Comment:
For creator så er kanskje pending tickets vel så relevant som nye?
```suggestion
    Pending: true,
```